.github/workflows/bootstrap.yaml

```diff
@@ -123,8 +123,10 @@ jobs:
         run: .github/workflows/scripts/install_python_client.sh
 
       - name: Before Script
-        run: |
-          .github/workflows/scripts/before_script.sh
+        run: .github/workflows/scripts/before_script.sh
+        shell: bash
+        env:
+          REDIS_DISABLED: ${{ contains('s3', matrix.env.TEST) }}
```

Comment:
I sense that this is not exactly the same as in the file that gets generated. (Not your fault!) Can you regenerate it please?
@fao do you see a way that we can asure this part of the file equals the on the fly generated bit by bit?

Reply:
I saw this tweet yesterday:
https://twitter.com/github/status/1461106904122417156?s=20
about reusable workflows, I didn't read the article, but maybe this is the solution

Reply:
No, the question was about comparing two files on the fly. We are not going to run another workflow right now.